.rustfmt.toml

```diff
@@ -1,17 +1,17 @@
-unstable_features = true
-
-blank_lines_lower_bound = 0
-condense_wildcard_suffixes = true
-error_on_line_overflow = true
-error_on_unformatted = true
-format_code_in_doc_comments = true
-format_macro_matchers = true
-format_strings = true
-imports_granularity = "Crate"
-normalize_comments = true
-normalize_doc_attributes = true
-reorder_impl_items = true
-group_imports = "StdExternalCrate"
-use_field_init_shorthand = true
-use_try_shorthand = true
-wrap_comments = true
\ No newline at end of file
+#unstable_features = true
```

Comment:
Why this is commented out?

Please adhere to defined formatting standards.

Reply:
Because using nightly formating didn't work as cargo +nightly fmt is currently bugged since the last nightly.

Reply:
cc @fominok